parse.y

```diff
@@ -2925,6 +2925,11 @@ rb_parser_ary_free(rb_parser_t *p, rb_parser_ary_t *ary)
                 | tCONSTANT
                 ;
 
+%rule %inline user_or_keyword_variable
+                : user_variable
```

Comment:
@ydah It seems that `ripper: var_field!` and `assignable` call is omitted from the rule whose RHS is `user_variable`, is it intentional?